test_runner/fixtures/neon_fixtures.py

```diff
@@ -947,8 +947,6 @@ def cleanup_local_storage(self):
                     continue
                 if SMALL_DB_FILE_NAME_REGEX.fullmatch(test_file.name):
                     continue
-                if FINAL_METRICS_FILE_NAME == test_file.name:
-                    continue
```

Comment:
I think we might need to keep that, if this comment is correct:
https://github.com/neondatabase/neon/blob/aa19f10e7e958fbe0e0641f2e8c5952ce3be44b3/test_runner/fixtures/paths.py#L183-L184